### PR TITLE
pdfslicer: init at 1.8.8

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/pdfslicer/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "pdfslicer";
+  version = "1.8.8";
+
+  src = fetchFromGitHub {
+    owner = "junrrein";
+    repo = "pdfslicer";
+    rev = "v${version}";
+    fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/pdfslicer/default.nix:23]: Can we unvendor the dependencies? The vendored ones are quite old.

**Comment by rmcgibbo** [comment]
This actually is a pretty nice tool. I built it on x86_64-linux and played around with the UI. Seems like a good addition that I would personally use from time to time.

**Comment by tomberek** [comment]
Result of `nixpkgs-review pr 115627` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pdfslicer</li>
  </ul>
</details>

File: pkgs/applications/misc/pdfslicer/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/applications/misc/pdfslicer/default.nix:23]: I want to try doing that, but I won't have time for a while.

Comment (dotlambda) [reply pkgs/applications/misc/pdfslicer/default.nix:23]: I tried this over at https://github.com/NixOS/nixpkgs/pull/116617.

**Pull request merged**